src/edu/washington/escience/myria/operator/TupleRangeSource.java

```diff
@@ -1,14 +1,15 @@
 package edu.washington.escience.myria.operator;
```

Comment:
This seems like a general refactoring suggestion,  I would like to finish up this PR and then focus on general refactoring.